core/templates/dev/head/domain/exploration/StateObjectFactory.js

```diff
@@ -18,40 +18,26 @@
  */
 
 oppia.factory('StateObjectFactory', [
-  'AnswerGroupObjectFactory',
-  function(AnswerGroupObjectFactory) {
+  'AnswerGroupObjectFactory', 'InteractionObjectFactory',
+  function(AnswerGroupObjectFactory, InteractionObjectFactory) {
   var State = function(name, content, interaction, paramChanges) {
     this.name = name;
+    this.classifier_model_id = null;
```

Comment:
Ooh sorry! I meant #2961. But you're right actually, this is my bad -- I was mixing up the two of these in my head.